app/controllers/service_requests_controller.rb

```diff
@@ -231,9 +199,8 @@ def confirmation
         send_epic_notification_for_user_approval(@protocol)
       end
     end
-
-    send_request_amendment_email_evaluation(previously_submitted_ssrs) unless previously_submitted_ssrs.empty?
-    send_confirmation_notifications(to_notify, send_request_amendment_and_not_initial) unless to_notify.empty?
+    NotifierLogic.new(@service_request, @sub_service_request, current_user).send_request_amendment_email_evaluation
+    NotifierLogic.new(@service_request, @sub_service_request, current_user).send_confirmation_notifications_submitted
```

Comment:
Is creating a new NotifierLogic object here necessary? Can you use the one you constructed on line 202?